### PR TITLE
Add method `Connection::ping()` to check server connection

### PR DESCRIPTION
**Pull request by hans-thomas**
Hi, due to the provided solution in #2649, I added a new method on the `Connection.php` to make it easy for users to access it.

### CODE REVIEW

File: src/Connection.php [hunk 1]
```diff
@@ -189,6 +193,21 @@ protected function createConnection(string $dsn, array $config, array $options):
...
 
+    /**
+     * Check the connection to the defined MongoDB host
+     *
+     * @return void
+     *
+     * @throws ConnectionException if connection to the server fails (for reasons other than authentication).
+     * @throws AuthenticationException if authentication is needed and fails.
+     * @throws RuntimeException if a server matching the read preference could not be found.
+     * @throws \MongoDB\Driver\Exception\InvalidArgumentException on argument parsing errors.
```
Comment (GromNaN) [inline src/Connection.php]: This method has no argument, this exception is not expected.

File: src/Connection.php [hunk 2]
```diff
@@ -189,6 +193,21 @@ protected function createConnection(string $dsn, array $config, array $options):
         return new Client($dsn, $options, $driverOptions);
     }
 
+    /**
+     * Check the connection to the defined MongoDB host
+     *
+     * @return void
```
Comment (GromNaN) [inline src/Connection.php]: Duplicate of the native return type. You can remove it.

**Review by GromNaN — COMMENTED** [review]
I think it's great, with a little cleanup.

**Review by GromNaN — APPROVED** [review]
LGTM.

**Review by alcaeus — APPROVED** [review]

File: tests/ConnectionTest.php [hunk 3]
```diff
@@ -231,4 +233,29 @@ public function testDriverName()
         $driver = DB::connection('mongodb')->getDriverName();
         $this->assertEquals('mongodb', $driver);
     }
+
+    public function testPingMethod()
+    {
+        $config = [
+            'name'     => 'mongodb',
+            'driver'   => 'mongodb',
+            'dsn'      => env('MONGODB_URI', 'mongodb://mongodb/'),
```
Comment (GromNaN) [inline tests/ConnectionTest.php]: To match the default configuration in other tests (see `tests/config/database.php`):
```suggestion
            'dsn'      => env('MONGODB_URI', 'mongodb://127.0.0.1/'),
```

File: src/Connection.php [hunk 4]
```diff
@@ -189,6 +193,18 @@ protected function createConnection(string $dsn, array $config, array $options):
         return new Client($dsn, $options, $driverOptions);
     }
 
+    /**
+     * Check the connection to the defined MongoDB host
```
Comment (GromNaN) [inline src/Connection.php]:
```suggestion
     * Check the connection to the MongoDB server
```

**Comment by GromNaN** [comment]
Thank you @hans-thomas

**Pull request merged**